Mathlib/Algebra/Polynomial/RingDivision.lean

```diff
@@ -750,6 +750,13 @@ theorem comp_eq_zero_iff : p.comp q = 0 ↔ p = 0 ∨ p.eval (q.coeff 0) = 0 ∧
       Or.rec (fun h => by rw [h, zero_comp]) (fun h => by rw [h.2, comp_C, h.1, C_0]) h
 #align polynomial.comp_eq_zero_iff Polynomial.comp_eq_zero_iff
 
+lemma aeval_ne_zero_of_isCoprime [CommSemiring R] [Nontrivial S] [Semiring S] [Algebra R S]
+    {p q : R[X]} (h : IsCoprime p q) (s : S) : ¬(aeval s p = 0 ∧ aeval s q = 0) := by
```

Comment:
Stating this (and the other lemmas) with an `Or` feels more reasonable to me; not-least because this spelling isn't simp-normal.

Reply:
I'll change to that, but will keep the naming. Note that this just adds a layer of `not_and_or` everywhere.